capstone/src/main/webapp/index.html

```diff
@@ -5,7 +5,14 @@
     <title>STEP Capstone</title>
     <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
+    <link
+      href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
   </head>
-  <body>
+  <body onload="getRestaurants();">
+    <div id="restaurant-space"></div>
```

Comment:
Does whatever framework you're using support templating? This kind of place would be a good place to use it.

Reply:
I believe Josh has some great progress on this/making a nice UI for the home page in his branch--I had mainly added this for my testing purposes!

Reply:
Ok cool, glad it's coming along!